src/test/kotlin/uk/gov/dluhc/printapi/service/PrintServiceTest.kt

```diff
@@ -32,18 +32,17 @@ class PrintServiceTest {
     @Test
     fun `should save send application to certificate`() {
         // Given
-        val ero = buildEroManagementApiEroDto()
-        val localAuthority = ero.localAuthorities[1]
-        val message = buildSendApplicationToPrintMessage(gssCode = localAuthority.gssCode)
+        val issuer = buildIssuerDto()
+        val message = buildSendApplicationToPrintMessage()
         val certificate = buildCertificate()
-        given(eroClient.getElectoralRegistrationOffices(any())).willReturn(ero)
-        given(certificateMapper.toCertificate(any(), any(), any())).willReturn(certificate)
+        given(eroClient.getIssuer(any())).willReturn(issuer)
+        given(certificateMapper.toCertificate(any(), any())).willReturn(certificate)
 
         // When
         printService.savePrintMessage(message)
 
         // Then
-        verify(eroClient).getElectoralRegistrationOffices(message.gssCode!!)
+        verify(eroClient).getIssuer(message.gssCode!!)
```

Comment:
Minor, @Neil-Massey . 
It will be good to verify the certificate mapper call as well.

<pre>
verify(certificateMapper).toCertificate(message, issuer)
</pre>